tests/cross_functional/kcs/test_selinux_relabel_solution.py

```
@@ -190,7 +195,9 @@ def get_pod_start_time(self, pod_name):
                 return time_difference.total_seconds()
 
         except CommandFailed as exc:
-            log.exception(f"Error retrieving pod information for '{pod_name}': {exc}")
+            logger.exception(
+                f"Error retrieving pod information for '{pod_name}': {exc}"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)